src/pal/src/debug/debug.cpp

```diff
@@ -343,15 +344,18 @@ DebugBreakCommand()
     const char *command_string = getenv (PAL_RUN_ON_DEBUG_BREAK);
     if (command_string) {
         char pid_buf[sizeof (PID_TEXT) + 32];
-        char exe_buf[sizeof (EXE_TEXT) + MAX_LONGPATH + 1];
+		    PathCharString exe_bufString;
```

Comment:
I don't think we have published explicit rules. The guidance is to use style that's similar to the code surrounding a change. But we don't use tabs in any of our sources, since it breaks formatting in different editors / code viewers.